docs/tutorials/services/data-mlmodel-tutorial.md

```diff
@@ -1,7 +1,6 @@
 ---
 title: "Capture Data and Train a Model"
 linkTitle: "Capture Data and Train a Model"
-weight: 4
```

Comment:
with this one we do want to keep it because it's one of the 4 we do want at the top. We just should probably discuss if we add or remove one of them